.palantir/revapi.yml

```diff
@@ -1,4 +1,85 @@
 acceptedBreaks:
+  "1.0.0":
+    org.apache.iceberg:iceberg-core:
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseMetadataTable"
+      new: "class org.apache.iceberg.BaseMetadataTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseTable"
+      new: "class org.apache.iceberg.BaseTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseTransaction.TransactionTable"
+      new: "class org.apache.iceberg.BaseTransaction.TransactionTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.MetricsConfig"
+      new: "class org.apache.iceberg.MetricsConfig"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.SerializableTable"
+      new: "class org.apache.iceberg.SerializableTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.TableMetadata"
+      new: "class org.apache.iceberg.TableMetadata"
+      justification: "Serialization changed"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.DELETE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.DELETE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.MERGE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.MERGE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.UPDATE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.UPDATE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.method.addedToInterface"
+      new: "method <T extends org.apache.iceberg.rest.RESTResponse> T org.apache.iceberg.rest.RESTClient::delete(java.lang.String,\
+        \ java.util.Map<java.lang.String, java.lang.String>, java.lang.Class<T>, java.util.Map<java.lang.String,\
+        \ java.lang.String>, java.util.function.Consumer<org.apache.iceberg.rest.responses.ErrorResponse>)"
+      justification: "Adding new method where query params can be used"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.hadoop.conf.Configuration org.apache.hadoop.conf.Configurable::getConf()\
+        \ @ org.apache.iceberg.hadoop.HadoopConfigurable"
+      justification: "New method was added to org.apache.hadoop.conf.Configurable"
```

Comment:
Why is this here? It doesn't seem to reference an Iceberg class?

Reply:
`org.apache.iceberg.hadoop.HadoopConfigurable` is the class in Iceberg and it extends `org.apache.hadoop.conf.Configurable`, but TBH I'm not sure why it even complains. 

Initially I thought that `org.apache.hadoop.conf.Configurable` was just a marker interface without any methods and that `getConf()` / `setConf` were added later, but that's not the case as those methods existed in the interface for [many years](https://github.com/apache/hadoop/blame/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configurable.java).

Reply:
It should be fine to suppress it, I guess.